opp_msgs/CMakeLists.txt

```diff
@@ -29,7 +29,9 @@ add_message_files(
 add_service_files(
   DIRECTORY srv
   FILES
-    GetROISelection.srv
+      GetROISelection.srv
+      GetPathSelectionMesh.srv
+      GetPathSelectionCloud.srv
```

Comment:
You might consider combining these messages if you don't need access to the triangle information from the mesh

Reply:
Connor is tasked with adding the cloud rather than mesh implementation. I just added the srv definition in anticipation of that implementation. We definitely use the triangle info from the mesh. Once the cloud version is working, it may be possible to treat meshes as clouds, but not yet.